GVFS/GVFS.FunctionalTests/GVFSTestConfig.cs

```diff
@@ -16,14 +17,23 @@ public static class GVFSTestConfig
 
         public static bool ReplaceInboxProjFS { get; set; }
 
+        public static bool StartedFromDebugger { get; set; }
+
         public static string PathToGVFS
         {
             get
             {
-                return
-                    TestGVFSOnPath ? 
-                    Properties.Settings.Default.PathToGVFS : 
-                    Path.Combine(Properties.Settings.Default.CurrentDirectory, Properties.Settings.Default.PathToGVFS);
+                if (TestGVFSOnPath)
+                {
+                    return Properties.Settings.Default.PathToGVFS;
+                }
+
+                if (StartedFromDebugger && RuntimeInformation.IsOSPlatform(OSPlatform.OSX))

```

Comment:
Couldn't you just check `Debugger.IsAttached` here? Why do we need a new property?

Reply:
> Couldn't you just check Debugger.IsAttached here?

If we did that then the tests would start looking in the wrong path if you attached the debugger to tests started from the command line.

> Why do we need a new property?

We need the property to remember that we were started from the debugger.

Reply:
That's a good point, but I think the best way to guarantee a stable path would be to compute it once and store the path itself.

Reply:
That's a good idea, I'll switch this to `{ get; set; }` and we can set it when the program starts.